### PR TITLE
renamed-common class name

### PR DESCRIPTION
**Pull request by mahnoorgul-deriv**
In this PR, I have renamed commonEuPage's class. From `FooterEu` to `commonPage`.  As this class not only contains footer elements, but all other elements have been added in it.

### CODE REVIEW

File: cypress/support/POM/commonPage.js [hunk 1]
```diff
@@ -184,10 +185,11 @@ class footerEuPage {
     footerYoutubeLogo: () => cy.findByRole('link', { name: '_t_Youtube_t_' }),
     footerLinkedInLogo: () => cy.findByRole('link', { name: '_t_Linkedin_t_' }),
   }
-  eUPDFs = {
-    EUlicencePDF: '/regulatory/Deriv_Investments_(Europe)_Limited.pdf',
-    EURiskDiscPDF: '/tnc/eu/risk-disclosure.pdf',
+  euPdfs = {
+    EulicencePdf: '/regulatory/Deriv_Investments_(Europe)_Limited.pdf',
```
Comment (meeyun-deriv) [inline cypress/support/POM/commonPage.js:189]: lets standardised the camel case, it should be `euLicencePdf`

**Review by meeyun-deriv — CHANGES_REQUESTED** [review]

File: cypress/support/POM/commonPage.js (hunk 1 above)
Comment (mahnoorgul-deriv) [reply cypress/support/POM/commonPage.js]: updated in recent PR.

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Comment by mahnoorgul-deriv** [comment]
Reviewed this PR, closing it for now as some changes are already made to code no need to merge this.

**Pull request closed without merging**